src/rp2_common/cyw43_driver/CMakeLists.txt

```diff
@@ -69,13 +90,16 @@ if (EXISTS ${PICO_CYW43_DRIVER_PATH}/${CYW43_DRIVER_TEST_FILE})
     target_link_libraries(cyw43_driver_picow INTERFACE
             ${CYW43_FIRMWARE_OBJ}
             )
+    target_include_directories(cyw43_driver_picow INTERFACE
+            $(CYW43_FIRMWARE_LOC) # contains a header file with firmware details
+            )
     target_link_libraries(cyw43_driver_picow INTERFACE
             cyw43_driver_base
             pico_stdlib
             hardware_pio
             hardware_dma
             hardware_exception
+            pico_uzlib
```

Comment:
Should `pico_uzlib` be included only if `${CYW43_DECOMPRESS_FIRMWARE}` is set? :shrug:

Reply:
I would expect it to have no effect if it's unused?